h2o-genmodel/src/main/java/hex/genmodel/algos/tree/SharedTreeMojoModel.java

```diff
@@ -337,9 +339,18 @@ private static void computeTreeGraph(SharedTreeSubgraph sg, SharedTreeNode node,
         if (!naVsRest) {
             // Extract value or group to split on
             if (equal == 0) {
+              float splitVal = ab.get4f();
+
+              if (domains[colId] != null) {
+                GenmodelBitSet bs = new GenmodelBitSet(0);
+                final ByteBuffer byteBuffer = ByteBuffer.allocate(4).order(ByteOrder.nativeOrder())
```

Comment:
This is just an early stage attempt, detailed description of how these bytes should look like is in `CompressedTree`.